frontend/src/colors.js

```diff
@@ -0,0 +1,40 @@
+// if adding to this, please run 'yarn makecolors' from the frontend dir
+// to rebuild the matching colors.scss file'
+const colors = {
```

Comment:
Would grouping these into sub objects provide any additional value? colors.ttahub.blue or color.text.link

Reply:
I don't see a benefit right now, but maybe you can expand on that?